<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolbarEditor</class>
 <widget class="QDialog" name="ToolbarEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>719</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QLabel" name="toolbarLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Toolbar</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="actionsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Actions</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="availableActionsLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Available Actions</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" rowspan="2" colspan="2">
    <widget class="QListWidget" name="toolbarList">
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QListWidget" name="actionList"/>
   </item>
   <item row="2" column="4" rowspan="2">
    <widget class="QListWidget" name="availableList"/>
   </item>
   <item row="0" column="0" colspan="5">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="workspaceLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Workspace:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="workspaceName">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="add">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons/go-previous.svg</normaloff>:/data/icons/go-previous.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="remove">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons/go-next.svg</normaloff>:/data/icons/go-next.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="up">
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons/arrow_up.svg</normaloff>:/data/icons/arrow_up.svg</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="down">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="musescore.qrc">
         <normaloff>:/data/icons/arrow_down.svg</normaloff>:/data/icons/arrow_down.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>workspaceName</tabstop>
  <tabstop>toolbarList</tabstop>
  <tabstop>actionList</tabstop>
  <tabstop>add</tabstop>
  <tabstop>remove</tabstop>
  <tabstop>up</tabstop>
  <tabstop>down</tabstop>
  <tabstop>availableList</tabstop>
 </tabstops>
 <resources>
  <include location="musescore.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ToolbarEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ToolbarEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>297</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
